src/reconstruct/dc_inline.hpp

```diff
@@ -0,0 +1,80 @@
+//========================================================================================
```

Comment:
Should we remove the old donor cell and PLM reconstructions and just use these?

Reply:
I'd keep them (in a Kokkosized version, which I already have in another repo so I can add those easily) in order to allow for more flexibility on the application side (if we only keep one then it'd force the application design to use nested  loops [or not] when employing those Parthenon functionality).